packages/grid/x-grid-modules/src/constants/cssClassesConstants.ts

```diff
@@ -2,5 +2,5 @@
 export const CELL_CSS_CLASS = 'material-cell';
```

Comment:
Some for all the other class names?

```suggestion
export const CELL_CSS_CLASS = 'MuiDataGrid-cell';
```